package.json

```diff
@@ -6,17 +6,20 @@
   "bugs": {
     "url": "https://github.com/blinkmobile/angular-location.js/issues"
   },
-  "dependencies": {},
+  "dependencies": {
+    "@blinkmobile/geolocation": "^2.0.0",
```

Comment:
My feeling is, in terms of a tree of dependencies, that leaves should specify strict version dependencies, whilst non-leaves can get away from less-strict version ranges.

I'm not sure this is something we've properly discussed and codified, however, so maybe we should have a chat with the team about pros and cons?